src/main/java/net/dries007/tfc/objects/items/ceramics/ItemJug.java

```diff
@@ -95,7 +95,7 @@ else if (!world.isRemote && jugCap.drain(CAPACITY, false) == null && rayTrace !=
                 }
                 else
                 {
-                    player.world.playSound(null, player.getPosition(), TFCSounds.JUG_BLOW, SoundCategory.PLAYERS, 1.0f, 1.0f);
+                    player.world.playSound(null, player.getPosition(), TFCSounds.JUG_BLOW, SoundCategory.PLAYERS, 1.0f, Constants.RNG.nextFloat() * 0.3f + 0.8f);
```

Comment:
This is not the player yaw - the intent is to make it make different sounds based on what direction you face up / down.